src/components/JobsTrainingButtons/index.jsx

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from "react";
+import { useTranslation } from "react-i18next";
+import "./index.scss";
+
+const JobsTrainingButtons = ({ setCardsArr, setSearchCards }) => {
+  const { t } = useTranslation();
+  const [isDisabled, setIsDisabled] = useState(true);
+  const buttonText = ["Job Opportunities", "Trainings / Workshops"];
```

Comment:
move below line 3, give a more informative name than buttonText

Reply:
as there appear to be multiple buttons. the one on line 10 is not using this button text.

Reply:
opportunityListBtns is using this buttonText, I changed the variable name and moved it above opportunityListBtns.